examples/music-licensing-challenge/src/app/graphql/queries.py

```
@@ -0,0 +1,52 @@
+from typing import List, Optional
+
+from strawberry import ID, field, type
```

Ruff (0.15.13)

[error] Import `type` is shadowing a Python builtin

(A004)